app/uk/gov/hmrc/payetaxcalculatorfrontend/views/quickcalc/quick_calc_form.scala.html

```diff
@@ -7,50 +7,58 @@
 
 @main_template(title = Messages("quick_calc_form.title"), mainClass = Some("full-width")) {
 
-  <div class="grid-layout">
-
-    <div class="grid-layout__column grid-layout__column--2-3">
-
-      <header class="page-header">
-        <h1>Question</h1>
-      </header>
-
-      <fieldset class="form-field-group js-visible js-aria-show js-toggle"
-                id="us-number"
-                data-trigger="js-toggle-trigger">
-        <label class="block-label" for="us-phone-number-toggle-open">Yes
-          <input type="radio"
-                 id="us-phone-number-toggle-open"
-                 class="js-toggle-trigger"
-                 data-target="us-phone-number-open-target"
-                 value="yes"
-                 name="us-number"
-                 required />
-        </label>
-        <div id="us-phone-number-open-target" class="panel-indent hidden">
-          <h2>Open</h2>
-          <p>Example paragraph text</p>
-        </div>
-        <label class="block-label" for="us-phone-number-toggle-close">No
-          <input type="radio"
-                 id="us-phone-number-toggle-close"
-                 class="js-toggle-trigger"
-                 data-target="us-phone-number-close-target"
-                 value="no"
-                 name="us-number"
-                 required/>
-        </label>
-        <div id="us-phone-number-close-target" class="panel-indent hidden">
-          <h2>Close</h2>
-          <p>Example paragraph text</p>
+    <div class="grid-layout">
+
+        <div class="grid-layout__column grid-layout__column--2-3">
+
+            <header class="page-header">
+                <h1>@Messages("quick_calc.about_tax_code.header")</h1>
+            </header>
+
+            <div class="form-group">
+                <fieldset
+                class="inline js-visible js-aria-show js-toggle"
+                data-trigger="js-toggle-trigger"
+                >
+
+                    <label class="block-label" for="tax-code-yes">@Messages("yes")
+                        <input
+                        type="radio"
+                        id="tax-code-yes"
+                        class="js-toggle-trigger"
+                        data-target="tax-code-yes-target"
+                        name="tax-code"
+                        value="yes"
+                        required />
+
+                    </label>
+                    <label class="block-label" for="tax-code-no">@Messages("no")
+                        <input
+                        type="radio"
+                        id="tax-code-no"
+                        class="js-toggle-trigger"
+                        name="tax-code"
+                        value="no"
+                        required />
+
+                    </label>
+                    <div class="panel-indent hidden" id="tax-code-yes-target">
+                        <label class="form-label" for="code-input">@Messages("quick_calc.about_tax_code.has_tax_code")</label>
```

Comment:
you have a for attribute that points to code-input but what the element's id is tax-code actually